buildSrc/src/main/groovy/nva.publication.channels.java.java-conventions.gradle

```diff
@@ -15,10 +15,16 @@ dependencies {
     implementation libs.bundles.nva
     implementation libs.bundles.logging
     implementation libs.bundles.jackson
-    implementation libs.guava
     implementation libs.commons.validator
     implementation libs.open.csv
     implementation libs.aws.sdk2.s3
+    implementation libs.aws.sdk2.dynamodb
+    implementation libs.aws.sdk2.dynamodb.enhanced
+
+    implementation(libs.dynamodDbLocal) {
```

Comment:
Should this be a `testImplementation`, to avoid shipping this to production?

Reply:
Nice catch, will fix it.